src/components/CardSelector/CardSelector.scss

```diff
@@ -1,21 +1,23 @@
+@use "../../tokens/borders/borders" as *;
```

Comment:
As you use CSS custom properties, you don't need to import them. They are global.
You only need to import variables with `@use` when you need SCSS variables like `$myvariable`.

Please remove all imports.